src/PrintLoopNest.cpp

```diff
@@ -99,6 +98,15 @@ class PrintLoopNest : public IRVisitor {
             out << " in [" << min_val << ", " << max_val << "]";
         }
 
+        if (op->device_api != DeviceAPI::None) {
+          if (op->device_api == DeviceAPI::Host) {
```

Comment:
I forget when none vs host is used. Does this change what is output for all code?